src/requirements/requirement-graph-builder.ts

```diff
@@ -102,10 +102,10 @@ const buildRequirementFulfillmentGraph = <
 
   // Phase 3: Respect user's choices on double-counted courses.
   userCourses.forEach(({ uniqueId }) => {
-    // uniqueId === -1 means it's AP/IB course.
+    // uniqueId < 0 means it's AP/IB course.
```

Comment:
Again could we specify that AP/IB course if uniqueId < -1 since it's swim test for uniqueId === -1